tests/acceptance/features/apiGraph/createGroup.feature

```diff
@@ -0,0 +1,38 @@
+@api @skipOnOcV10
+Feature: create group
+  Only user with admin permissions can create new groups
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And the administrator has given "Alice" the role "Admin" using the settings api
+
+
+  Scenario Outline: admin user creates a group
+    When user "Alice" creates a group "<groupname>" using the Graph API
+    And the administrator creates a group "grp1" using the Graph API
+    Then the HTTP status code should be "200"
+    And group "<groupname>" should exist
+    Examples:
+    | groupname       |
+    | simplegroup     |
+    | España§àôœ€     |
+    | नेपाली            |
+    | $x<=>[y*z^2+1]! |
+    | 😅 😆           |
+    | comma,grp1      |
+    | Finance (NP)    |
+    | slash\Middle    |
+
+
+  Scenario: admin user tries to create a group that already exists
+    Given group "mygroup" has been created
+    When user "Alice" tries to create a group "mygroup" using the Graph API
```

Comment:
```suggestion
    When user "Alice" creates a group "mygroup" using the Graph API
```
The scenarios title itself says admin user tries to create group. change if it seems feasible.

Reply:
`tries to` seems good to me for the negative assertion tests like this.